src/ui/pages/FullPageScanner/FullPageScanner.test.tsx

```diff
@@ -13,31 +18,41 @@ import { connectionsFix } from "../../__fixtures__/connectionsFix";
 import { OperationType } from "../../globals/types";
 import { FullPageScanner } from "./FullPageScanner";
 
-const startScan = jest.fn(
-  (args: any) =>
-    new Promise((resolve) => {
-      setTimeout(() => {
-        resolve({
-          hasContent: true,
-          content:
+const addListener = jest.fn(
+  (eventName: string, listenerFunc: (result: BarcodeScannedEvent) => void) => {
+    setTimeout(() => {
+      listenerFunc({
+        barcode: {
+          displayValue:
+            "http://dev.keria.cf-keripy.metadata.dev.cf-deployments.org/oobi?groupId=72e2f089cef6",
+          format: BarcodeFormat.QrCode,
+          rawValue:
             "http://dev.keria.cf-keripy.metadata.dev.cf-deployments.org/oobi?groupId=72e2f089cef6",
-        });
-      }, 100);
-    })
+          valueType: BarcodeValueType.Url,
+        },
+      });
+
+      return {
+        remove: jest.fn(),
+      };
+    }, 100);
+  }
 );
 
-jest.mock("@capacitor-community/barcode-scanner", () => {
+jest.mock("@capacitor-mlkit/barcode-scanning", () => {
   return {
-    ...jest.requireActual("@capacitor-community/barcode-scanner"),
+    ...jest.requireActual("@capacitor-mlkit/barcode-scanning"),
     BarcodeScanner: {
-      checkPermission: () =>
+      checkPermissions: () =>
         Promise.resolve({
-          granted: true,
+          camera: "granted",
```

Comment:
thanks for fixing tests @Sotatek-DukeVu - @sdisalvo-crd we could add a small task to cover the scenario where it's not granted and we try to call `requestPermissions` (in the previous plugin, these were called together in one go but now separate)

I wouldn't cover this now though not important - we can do it next week or week after

Reply:
@iFergal  @sdisalvo-crd All tests are fixing. Please help me review and merge.
Thanks.

Reply:
Sure @iFergal I already have a ticket for improving unit tests, I will add this task to [that ticket](https://cardanofoundation.atlassian.net/browse/DTIS-1255) 👍